csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -3692,4 +3692,291 @@ Zach | Turk | Microsoft
+  * `/product_tree/relationships[]/full_product_name/product_identification_helper/serial_numbers[]`
+  * `/product_tree/relationships[]/product_reference`
+  * `/product_tree/relationships[]/relates_to_product_reference`
+  * `/vulnerabilities[]/acknowledgments[]/names[]`
+  * `/vulnerabilities[]/acknowledgments[]/organization`
+  * `/vulnerabilities[]/cve`
+  * `/vulnerabilities[]/cwe/id`
+  * `/vulnerabilities[]/cwe/name`
+  * `/vulnerabilities[]/notes[]/audience`
+  * `/vulnerabilities[]/notes[]/title`
+  * `/vulnerabilities[]/product_status/first_affected[]`
+  * `/vulnerabilities[]/product_status/first_fixed[]`
+  * `/vulnerabilities[]/product_status/fixed[]`
+  * `/vulnerabilities[]/product_status/known_affected[]`
+  * `/vulnerabilities[]/product_status/known_not_affected[]`
+  * `/vulnerabilities[]/product_status/last_affected[]`
+  * `/vulnerabilities[]/product_status/recommended[]`
+  * `/vulnerabilities[]/product_status/under_investigation[]`
+  * `/vulnerabilities[]/remediations[]/group_ids[]`
+  * `/vulnerabilities[]/remediations[]/product_ids[]`
+  * `/vulnerabilities[]/scores[]/cvss_v2/vectorString`
+  * `/vulnerabilities[]/scores[]/cvss_v3/vectorString`
+  * `/vulnerabilities[]/scores[]/products[]`
+  * `/vulnerabilities[]/threats[]/group_ids[]`
+  * `/vulnerabilities[]/threats[]/product_ids[]`
+  * `/vulnerabilities[]/title`
+
+* 10 000 for
+  * `/document/acknowledgments[]/summary`
+  * `/document/distribution/text`
+  * `/document/publisher/contact_details`
+  * `/document/publisher/issuing_authority`
+  * `/document/references[]/summary`
+  * `/document/tracking/revision_history[]/summary`
+  * `/product_tree/branches[]/product/product_identification_helper/cpe`
+  * `/product_tree/branches[]/product/product_identification_helper/purl`
+  * `/product_tree/branches[](/branches[])*/product/product_identification_helper/cpe`
+  * `/product_tree/branches[](/branches[])*/product/product_identification_helper/purl`
+  * `/product_tree/full_product_names[]/product_identification_helper/cpe`
+  * `/product_tree/full_product_names[]/product_identification_helper/purl`
+  * `/product_tree/product_groups[]/summary`
+  * `/product_tree/relationships[]/full_product_name/product_identification_helper/cpe`
+  * `/product_tree/relationships[]/full_product_name/product_identification_helper/purl`
+  * `/vulnerabilities[]/acknowledgments[]/summary`
+  * `/vulnerabilities[]/involvements[]/summary`
+  * `/vulnerabilities[]/references[]/summary`
+  * `/vulnerabilities[]/remediations[]/entitlements[]`
+
+* 30 000 for
+  * `/document/notes[]/text`
+  * `/vulnerabilities[]/notes[]/text`
+
+* 250 000 for
+  * `/vulnerabilities[]/remediations[]/details`
+  * `/vulnerabilities[]/remediations[]/restart_required/details`
+  * `/vulnerabilities[]/threats[]/details`
+
+## URI length
+
+A string with format `uri` should not have a length greater than 20000. This applies to:
```

Comment:
Most browsers have a limit of 2,000 characters on URLs, so 20,000 seems very generous.

Reply:
Well, that is a URI :wink: not necessarily a URL ... but, the limits we will publish after discussing specific first (possibly other) limits will be the real thing and gain of this nice proposal from @tschmidtb51 ... :+1:

Reply:
True, but if you look at the list of attributes below, more than a third are URLs even though the type of the field is technically `uri`.